transformer_engine/pytorch/attention.py

```diff
@@ -381,6 +385,11 @@ def get_attention_backend(
 
     # Filter: Execution type
     if fp8 and fp8_meta["recipe"].fp8_dpa:
+        if use_flash_attention and not _flash_attn_3_plus:
+            logger.debug(
+                "Disabling FlashAttention as FlashAttention 3 is not available for FP8 DPA/FP8 MHA."
```

Comment:
Maybe something like this? "Disabling FlashAttention as FlashAttention 2 does not support FP8 DPA/FP8 MHA."